fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraAclIT.java

```diff
@@ -185,6 +187,32 @@ public void testGetNonExistentAcl() throws Exception {
 
     }
 
+    @Test
+    public void testGetDefaultRootAcl() throws Exception {
+        final String rootAclUri = serverAddress + FCR_ACL;
+        try (final CloseableDataset dataset = getDataset(new HttpGet(rootAclUri))) {
+            final DatasetGraph graph = dataset.asDatasetGraph();
+            assertTrue(graph.contains(ANY,
+                                      createURI(rootAclUri),
+                                      createURI("http://www.w3.org/2000/01/rdf-schema#label"),
+                                      createLiteral("Root Authorization")));
```

Comment:
This test would also pass if the `test-root-authorization.ttl` had been returned. Maybe change the label text in `test-root-authorization.ttl` so that it is not a superset of the default authorization.

Reply:
@awoods I'll change the label text to something like `(Test) root ACL` anyway. But I think it should comparing the whole string for a match so it doesn't matter. I tested it with test-root-authorization.ttl by adding the line `System.setProperty(ROOT_AUTHORIZATION_PROPERTY, "./target/test-classes/test-root-authorization.ttl")` at the beginning of the test method before line https://github.com/fcrepo4/fcrepo4/pull/1387/files/db74a9a3142792328c808f63996246f32d6573ff#diff-2e4f22c10f7216f71eaec0d2e3a6fe93R192, I see the test failed.